test/functional/feature_notifications.py

```diff
@@ -13,13 +13,19 @@
     connect_nodes,
 )
 
+# Linux allow all characters other than \x00
+# Windows disallow control characters (0-31) and /\?%:|"<>
+FILE_CHAR_START = 32 if os.name == 'nt' else 1
+FILE_CHAR_END = 128
+FILE_CHAR_BLACKLIST = '/\\?%*:|"<>' if os.name == 'nt' else '/'
```

Comment:
Would be nice to autodetect this in the test.

Reply:
> What about macOS, BSD, Amiga, etc?

> Would be nice to autodetect this in the test.

It shouldn't be too hard to autodetect by just writing to files and seeing what succeeds, but in the worst case this test just doesn't work on an obscure platform and the fix is trivial, so I don't think it's is a big deal